pennylane/operation.py

```diff
@@ -1083,7 +1083,13 @@ def grad_method(self):
 
         Default is ``'F'``, or ``None`` if the Operation has zero parameters.
         """
-        return None if self.num_params == 0 else "F"
+        if self.num_params == 0:
+            return None
+        try:
+            self.parameter_frequencies  # pylint:disable=pointless-statement
+            return "A"
+        except OperatorPropertyUndefined:
+            return "F"
```

Comment:
not for this PR, but I would like to remove this method if possible

Reply:
I feel like it is up to the gradient transform or the user to infer/determine the gradient method :)

Reply:
ticket created.